ql/src/test/queries/clientpositive/perf/cbo_query49.q

```diff
@@ -45,10 +45,10 @@ select
  union
  select 
  'catalog' as channel
- ,catalog.item
- ,catalog.return_ratio
- ,catalog.return_rank
- ,catalog.currency_rank
+ ,`catalog`.item
```

Comment:
This is a backward incompatible change. Could we make the `catalog` a non-reserved keyword?

Reply:
Make sense, done